tests/testthat/test-drive_ls.R

```diff
@@ -0,0 +1,34 @@
+context("List files")
+
+nm_ <- nm_fun("-TEST-drive-ls")
+
+run <- FALSE
+clean <- FALSE
+if (run) {
+  ## make sure directory is clean
+  if (clean) {
+    del_pths <- c(nm_("foo"), nm_("this-should-not-exist"))
+    del <- purrr::map(del_pths, drive_delete, verbose = FALSE)
```

Comment:
Isn't `drive_delete()` vectorized? If so, all the deletions in test set up should rely on that, rather than `purrr::map()`.